.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+
+  # GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    reviewers:
+      - "elastic/observablt-ci"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "22:00"
```

Comment:
Is the reason to specify Sunday and a time so that all `package-ecosystem: "github-actions"` dependabot PRs across Obs repos tend to be at the same time?

Reply:
to help with knowing mondays are the days for reviewing those dependencies